interview-questions.md

```diff
@@ -9,59 +9,82 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: They keyword `this` pertains to a specific object that is instantiated from a class. Using the keyword `this` refers to the object and the object's own unique properties that may or may not be similar to other objects instantiated from the same class.
 
-  Researched answer:
+  Researched answer: `this` is a JavaScript keyword that references an object. Specifically an object that is derived from a certain class.
```

Comment:
Make sure you are not copying and pasting but you are correct 'this' refers to the object it is inside of. The word that helps this concept along is instantiation. This refers to an object that is an instance of a class. 'This' is dynamic where each object will have some kind of identifier.

Reply:
'This' operates like a parameter.